index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<head>
+
+<link rel="stylesheet" href="application.css"></link>
+<link rel="stylesheet" href="equalizer.css"></link>
```

Comment:
What you should actually do is import just the `application.css` and inside that file, you will import all the other SCSS:

```scss
@import 'equalizer';
@import 'variables/colors'; // This file will contain all the colors of your page
@import 'styles.scss'; // The actual style sheet
```